nvdaHelper/archBuild_sconscript

```diff
@@ -147,6 +146,11 @@ if 'RTC' in debug:
 env.Append(PDB='${TARGET}.pdb')
 env.Append(LINKFLAGS='/OPT:REF') #having symbols usually turns this off but we have no need for unused symbols
 
+win10env = env.Clone()
+# Add C++ co-routine support

```

Comment:
Please add "for winrt usage" or something more specific. I haven't yet done a thorough review of this so I'm not sure I can make a precise suggestion yet. Generally, if I encountered this while trying to split up the code in this DLL or merge with another, or perhaps copy the approach for a new DLL, then I'd want to know which code / features depend on this flag and why it is necessary.